src/modules/orders/ShoppingDetails.tsx

```diff
@@ -8,143 +8,172 @@ import { useForm } from 'react-hook-form';
-            }}
-          >
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.City')}
-              variant='filled'
-              {...register('city')}
-            />
-            {errors.city && <p>{errors.city.message}</p>}
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.Street')}
-              variant='filled'
-              {...register('street')}
-            />
-            {errors.street && <p>{errors.street.message}</p>}
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.Building Number')}
-              variant='filled'
-              type='number'
-              {...register('buildingNumber')}
-            />
-            {errors.buildingNumber && <p>{errors.buildingNumber.message}</p>}
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.Floor')}
-              variant='filled'
-              type='number'
-              {...register('floor')}
-            />
-            {errors.floor && <p>{errors.floor.message}</p>}
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.Apartment Number')}
-              variant='filled'
-              type='number'
-              {...register('apartmentNumber')}
-            />
-            {errors.apartmentNumber && <p>{errors.apartmentNumber.message}</p>}
-            <TextField
-              id='filled-basic'
-              fullWidth
-              label={t('order.Last Name')}
-              variant='filled'
-              {...register('lastName')}
-            />
-            {errors.lastName && <p>{errors.lastName.message}</p>}
-            <Button type='submit' variant='contained' color='primary'>
-              {t('order.Save Details')}
-            </Button>
-          </Box>
-        )}
-      </FormControl>
-    </form>
-  );
-};
-
-export default ShoppingDetails;
+
+    const handleChange = (e: { target: { value: React.SetStateAction<string>; }; }) => {
+        setSelectedOption(e.target.value);
+    };
+
+    useEffect(() => {
+        getAllCart();
+    }, []);
+
+    const saveDetails = async (data: Record<string, any>) => {
+        try {
+            const newData = {
+                destinationAddress: {
+                    ...data
+                },
+                user: userId,
+                products: productsCart,
+                businessCode: currentEmployee.businessId,
+                settingManeger: 1,
+                deliveryMethod: selectedOption,
+            }
+            await addItem("orders", newData);
+            reset()
+            alert("Order saved successfully");
+        } catch (err) {
+            console.log(err);
+            showErrorToast("Error saving the order");
```

Comment:
נא לתרגם

Reply:
בנוסף, בכל מקום שיש alert תוסיפי הערה של todo: change this to use toast from storybook